HUD/SetWarningMessage.md

```diff
@@ -5,87 +5,71 @@ ns: HUD
+| 1            | OK          | INPUT_FRONTEND_ACCEPT | None |
+| 2            | Yes          | INPUT_FRONTEND_ACCEPT | None |
+| 3            | Back          | INPUT_FRONTEND_CANCEL | None |
+| 4            | Cancel          | INPUT_FRONTEND_CANCEL | None |
+| 5            | No          | INPUT_FRONTEND_CANCEL | None |
+| 6            | Retry          | INPUT_FRONTEND_X | None |
+| 7            | Restart          | INPUT_FRONTEND_X | None |
+| 8            | Skip         | INPUT_FRONTEND_X | None |
+| 9            | Quit         | INPUT_FRONTEND_CANCEL | None |
+| 10            | Adjust         | INPUTGROUP_FRONTEND_DPAD_LR | Adjust with left/right keyboard arrow icons |
+| 11            | Ignore          | INPUT_FRONTEND_X | None |
+| 12           | Share          | INPUT_FRONTEND_X | None |
+| 13           | Log In          | INPUT_FRONTEND_X | None |
+| 14            | Continue          | INPUT_FRONTEND_ACCEPT | None |
+| 15            | Adjust          | None | Adjust with left/right arrow icons |
+| 16            | Scroll          | None | Sets scroll button with up and down arrow icons |
+| 17            | Overwrite          | INPUT_FRONTEND_ACCEPT | None  |
+| 18            | Rockstar Games Sign Up          | INPUT_FRONTEND_ACCEPT | None  |
+| 19            | Confirm          | INPUT_FRONTEND_ACCEPT | None  |
+| 20            | Queue          | INPUT_FRONTEND_ACCEPT | None  |
+| 21            | Retry          | INPUT_FRONTEND_ACCEPT | None  |
+| 22            | Back          | INPUT_FRONTEND_CANCEL | Same as button index #3 but plays select sound instead of cancel.  |
+| 23            | Rockstar Games          | INPUT_FRONTEND_ACCEPT | This button\'s text label is `IB_UPDATE_SOCIAL_CLUB`, so it's used for updating the game.  |
+| 24            | Spectate          | INPUT_FRONTEND_X | None  |
+| 25            | OK          | INPUT_FRONTEND_CANCEL | OK with CANCEL instead of ACCEPT.  |
+| 26            | Cancel Transfer          | INPUT_FRONTEND_CANCEL |  |
+| 27            | Nothing          | None | Sets the loading spinner icon.  |
+| 28            | No - Return to Grand Theft Auto V         | INPUT_FRONTEND_CANCEL | None |
+| 29            | Cancel          | INPUT_FRONTEND_CANCEL | Same as #4 |
+| 30            | None          | None | Reserved for NO SOUND. |
+| 31            | Exit         | INPUT_FRONTEND_CANCEL | None |
+| 32            | No          | INPUT_FRONTEND_X | None |
+| 33            | Host          | INPUT_FRONTEND_CANCEL | None |
+| 34            | Search For Job          | INPUT_FRONTEND_ACCEPT | None  |
+| 35            | Nothing          | INPUT_FRONTEND_ACCEPT | None  |
+| 36            | Freemode          | INPUT_FRONTEND_ACCEPT | None  |
+| 37            | Submit          | INPUT_FRONTEND_ACCEPT | None  |
+| 38            | View Player Profile          | INPUT_FRONTEND_ACCEPT | None  |
 
 **Result of the example code:**
 [https://i.imgur.com/imwoimm.png](https://i.imgur.com/imwoimm.png)
```

Comment:
Can you delete that please? This is old and shouldn't been there in the first place. Also not working.

Reply:
Sure! The link was working when i made this commit which is weird